{{cookiecutter.app_name}}/tests/test_models.py

```diff
@@ -64,3 +64,13 @@ def test_roles(self):
         user.roles.append(role)
         user.save()
         assert role in user.roles
+
+    def test_roles_repr(self):
+        """Check __repr__ output for Role."""
+        role = Role(name="user")
+        assert role.__repr__() == "<Role(user)>"
+
+    def test_repr(self):
```

Comment:
Should this be
```suggestion
    def test_user_repr(self):
```
instead?